util/setting_test.go

```diff
@@ -0,0 +1,26 @@
+package util
+
+import (
+	"github.com/stretchr/testify/assert"
```

Comment:
Format this import.

Reply:
I thinks this is a problem github, look these pull requests:
https://github.com/casdoor/casdoor/pull/407/files
https://github.com/casdoor/casdoor/pull/432/files

This format is a same

Reply:
I didn't know about this convention.